lldb/source/DataFormatters/TypeSummary.cpp

```diff
@@ -116,6 +116,9 @@ std::string StringSummaryFormat::GetDescription() {
   return std::string(sstr.GetString());
 }
 
+ConstString StringSummaryFormat::GetName() { return ConstString(m_format_str); }
+ConstString StringSummaryFormat::GetImplType() { return ConstString("string"); }
```

Comment:
We should switch to `std::string` instead of `ConstString.. These objects aren't starting out as ConstString objects. We only want to use ConstString objects for things we are searching for by name where there will be many copies of the string.